### PR TITLE
Standardization of GTPP Chemical Recipes

### PR DESCRIPTION
**Pull request by GlodBlock**

### CODE REVIEW

File: src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java [hunk 1]
```diff
@@ -417,7 +417,7 @@ private static void chemplantRecipes() {
 
 					},
 					new FluidStack[] {
-							FluidUtils.getHydrofluoricAcid(4000)
+							FluidUtils.getHydrofluoricAcid(4500)
```
Comment (miozune) [inline src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java:421]: any reason?

File: src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java [hunk 2]
```diff
@@ -1954,25 +1931,26 @@ private static void chemicalReactorRecipes() {
...
 		GT_Values.RA.addChemicalRecipe(
-				ItemUtils.getSimpleStack(AgriculturalChem.mSodiumCarbonate, 1),
+				ItemUtils.getSimpleStack(AgriculturalChem.mSodiumCarbonate, 6),
 				CI.getNumberedAdvancedCircuit(1), 
-				Materials.NitricAcid.getFluid(1000),
-				GT_Values.NF, 
-				MISC_MATERIALS.SODIUM_NITRATE.getDust(1), 
+				Materials.NitricAcid.getFluid(2000),
+				Materials.CarbonDioxide.getGas(1000),
+				MISC_MATERIALS.SODIUM_NITRATE.getDust(5),
```
Comment (miozune) [inline src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java]: 10x

File: src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java [hunk 3]
```diff
@@ -1954,25 +1931,26 @@ private static void chemicalReactorRecipes() {
 				ItemUtils.getItemStackOfAmountFromOreDict("dustSmallSulfur", 8), 
 				20 * 20);		
 
+		//K + HNO3 = KNO3 + H
```
Comment (miozune) [inline src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java]: KOH + HNO3 = KNO3 + H2O

File: src/main/java/gtPlusPlus/plugin/agrichem/BioRecipes.java [hunk 4]
```diff
@@ -968,30 +966,30 @@ private static void recipeAluminiumPellet() {
 				ItemUtils.getSimpleStack(AgriculturalChem.mAluminiumPellet, 4),
 				20 * 30, 
 				64);
-
-
 	}
 
 	private static void recipeAlumina() {
+		//2NaAlO2 + 2NaOH + 2CO2 = Al2O3 + 2NaCO3 + H2O
```
Comment (miozune) [inline src/main/java/gtPlusPlus/plugin/agrichem/BioRecipes.java]: nit: 2Na2CO3

File: src/main/java/gtPlusPlus/xmod/gregtech/loaders/recipe/RecipeLoader_GlueLine.java [hunk 5]
```diff
@@ -93,63 +83,68 @@ private static void chemicalPlantRecipes() {
...
+                        ItemUtils.getSimpleStack(AgriculturalChem.mSodiumCarbonate, 6),
+                        MISC_MATERIALS.SODIUM_CYANIDE.getDust(3)
                 },
                 new FluidStack[] {
                         MISC_MATERIALS.CHLOROACETIC_ACID.getFluidStack(1000),
                         FluidUtils.getFluidStack("hydrochloricacid_gt5u", 2000)
                 },
                 new ItemStack[] {
-                        MISC_MATERIALS.CYANOACETIC_ACID.getDust(6)
+                        MISC_MATERIALS.CYANOACETIC_ACID.getDust(18),
```
Comment (miozune) [inline src/main/java/gtPlusPlus/xmod/gregtech/loaders/recipe/RecipeLoader_GlueLine.java]: 9x

**Review by miozune — COMMENTED** [review]
- 2-Ethylanthraquinone recipes
  - 1000mB Phthalic Acid = 15x Phthalic Anhydride (Phthalic Acid is integrated with GT one so we can't make it 144*15mB I guess)
  - 15x Phthalic Anhydride + 1000mB Ethylbenzene = 1000mB 2-Ethylanthraquinone (C6H4(CO)2O + C6H5CH2CH3 = C6H4(CO)2C6H3CH2CH3 + H2O)
  - 1000mB 2-Ethylanthraquinone + 2000mB Hydrogen -> 1000mB 2-Ethylanthrahydroquinone (C6H4(CO)2C6H3CH2CH3 + 2H = C6H4(COH)2C6H3CH2CH3)
  - 5000mB 2-Ethylanthrahydroquinone + 20000mB Air + 50mB Anthracene = 5000mB Hydrogen Proxide + 5000mB 2-Ethylanthraquinone (C6H4(COH)2C6H3CH2CH3 + 2O =(C6H4CH)2= H2O2 + C6H4(CO)2C6H3CH2CH3)
- 23x Borax + 2000mB Hydrochloric Acid = 4000mB Boric Acid + 4x Salt + 5000mB Water (Na2B4O7·10H2O + 2HCl = 4B(OH)3 + 2 NaCl + 5 H2O)
- Solar Salt recipes
  - 10x Potassium Nitrate + 15x Sodium Nitrate = 5000mB Solar Salt (Cold) (2KNO3 + 3NaNO3 = (KNO3)2(NaNO3)3)
  - 5000mB Solar Salt (Cold) = 10x Potassium Nitrate + 15x Sodium Nitrate ((KNO3)2(NaNO3)3 = 2KNO3 + 3NaNO3)

File: src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java (hunk 1 above)
Comment (GlodBlock) [reply src/main/java/gtPlusPlus/core/recipe/RECIPES_GREGTECH.java:421]: alk said the industrial hydrofluoric acid is equal to 2x hydrofluoric acid. https://github.com/GTNewHorizons/GTplusplus/pull/140#issuecomment-1052603231

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**